clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -31,19 +47,24 @@ case object GetServerVersion extends ClioCommand
 
 // WGS-uBAM commands.
 
-@CommandName(ClioCommand.getWgsUbamSchemaName)
-case object GetSchemaWgsUbam extends ClioCommand
+@CommandName(ClioCommand.getSchemaPrefix + WgsUbamIndex.commandName)
```

Comment:
Why not still `ClioCommand.getWgsUbamSchemaName`? Similarly for the rest.

Reply:
I'll change this back, I was trying out getting rid of the `ClioCommand` members for each command but in the end it didn't seem worthwhile (yet). One can imagine a future version of the code where commands and index names are passed separately as command line arguments, and the command line parser is initialized programmatically.